clash-protocols-memmap/src/Protocols/MemoryMap/Registers/WishboneStandard.hs

```diff
@@ -469,7 +470,20 @@ registerWbDf clk rst regConfig resetValue =
               , (pure resetValue, busActivity)
               )
  where
-  needReverse = ?busByteOrder /= ?regByteOrder
+  regByteOrder = ?regByteOrder
```

Comment:
Is there a reason that this introduces a new binder? AFAIK I only see it used in `needReverse` right below, the rest is using the implicit?

Reply:
Ahh right, I missed that in the quasiquote message. It's fine then, might be nice to move it closer to where the "issue" is, but that makes sense